apiserver/facades/client/machinemanager/machinemanager_test.go

```diff
@@ -318,6 +318,18 @@ func (s *MachineManagerSuite) TestUpdateMachineSeriesPermissionDenied(c *gc.C) {
 	c.Assert(err, gc.ErrorMatches, "permission denied")
 }
 
+func (s *MachineManagerSuite) TestUpgradeSeriesPrepareLocksMachine(c *gc.C) {
+	apiV4 := machinemanager.MachineManagerAPIV5{MachineManagerAPI: s.api}
+	machineTag := names.NewMachineTag("0")
+	_, err := apiV4.UpgradeSeriesPrepare(
+		params.UpdateSeriesArg{
+			Entity: params.Entity{
+				Tag: machineTag.String()},
+		},
+	)
+	c.Assert(err, jc.ErrorIsNil)
```

Comment:
add check to see if machine.CreateUpgradeSeriesPrepareLock is called.

Reply:
Here, I would l just like to see if the API call succeeds. I doesn't actually return anything yet although I am sure it will in the future. When that happens I will test against that returned value. Right now, all I would like to do is test that it doesn't error out. I generally shy away for using mocks in such a way that I am testing implementation vs behavior although I do understand that in some cases there is really no other choice. Here I believe that there are alternatives.

Reply:
Could you please add some comments to that affect then?